src/transformers/trainer.py

```diff
@@ -1100,8 +1103,16 @@ def get_optimizer_cls_and_kwargs(args: TrainingArguments) -> Tuple[Any, Any]:
                 elif "lion" in args.optim:
                     optimizer_cls = Lion
                     additional_optim_kwargs = {"betas": (args.adam_beta1, args.adam_beta2)}
+                elif "rmsprop" in args.optim:
+                    optimizer_cls = RMSprop
+                    # Above we pass all `adam_kwargs` to the optimizer, here
+                    # we only pass `optim_args` which can be passed to the user.
```

Comment:
Perhaps? Or are we passing these arguments back to the user? 

```suggestion
                    # we only pass `optim_args` which can be passed by the user.
```

Reply:
Indeed this is correct! 🙏 Thanks !